### DISCUSSION TITLE
Enhancement in the Greetings workflow for the New Comers

### DISCUSSION THREAD

**Discussion by IIITM-Jay** [IIITM-Jay/Property-management-System#31] [Ideas]
This discussion is open to all the new comers. They can share their ideas in a way to greet the new comers with pleasant and warm welcome so that they will enjoy their own contributions and the project as well.

Once again welcome to the Property management System :houses: 

I want everyone of us to be in loop and do share this repository to all your friends and colleagues so to have a feeling of team rather than a manager owning a repo or else, 'Team work is less "ego" and more "we go" ' and discuss the matters all round as well as doing PR's are opening issues as well.

Thank you.
-Jay

**Comment by akrish4** [comment]
### This is the current message we get when one opens a issue 
`issue-message: 'Welcome to the Property Management System :houses: '`
### I would like to change it to like this 
` 'Welcome to the Property Management System :houses: '`
`Thank you and congrats🎉 for opening your very first issue in this project. 
  Please adhere to our [Code of Conduct](file link).
  Please make sure not to start working on the issue, unless you get assigned to it.😄  `

**Reply by akrish4** [reply]
let me know if you like I will make the changes and open a pr , pls

**Comment by IIITM-Jay** [comment]
Anantha @akrish4 , I will look into the discussions, issues as well as PR's after two- three days with patience as your work should also be acknowledged fairly. Actually two-three days are going to be very busy for me hence I request.

I can suggest one thing now, that please remove the second line as it looks somewhat more restricted as we want the new comers to enjoy this project. Meanwhile you can also join some of your colleagues and juniors to have some discussion on this ideas because I have assigned you the lead investigatior of this issue and I would love to see you as mentoring your juniors as well.
As a DWoC , you will be working as a contributor but here you have amazing experience of doing both the contributions as well as some flavor of mentoring. :+1: 

Good Work :1st_place_medal:

**Reply by akrish4** [reply]
okay thats fine , I actually like to complete the work asap and look into other issues , so pls kindly check it so that I can make the pr , you cn take 3 or 4 days for reviewing it

**Comment by akrish4** [comment]
### I would like to change it to like this 
` 'Welcome to the Property Management System :houses: '`
`Please make sure not to start working on the issue, unless you get assigned to it.😄  `

### This would be the PR message 
`Congrats🎉 for opening your first pull request.`
`🙌 We will get back to you as soon as we can 😄.`

**Reply by akrish4** [reply]
Let me know if anything to be changed ?

**Comment by IIITM-Jay** [comment]
Please keep the followings:
`Welcome to the Property Management System :houses: '
Thank you and congrats🎉 for opening your very first issue in this project. Please adhere to our [Code of Conduct](file link).

And remove this one: (not required)
~Please make sure not to start working on the issue, unless you get assigned to it.😄~

**Reply by akrish4** [reply]
yeah okay , noted 
let me know if any changes needed in the above  pr message
